src/Core/Components/MenuButton/FluentMenuButton.razor

```diff
@@ -2,7 +2,7 @@
 @using Microsoft.FluentUI.AspNetCore.Components.DesignTokens
 @inherits FluentComponentBase
 
-<div class="fluent-menubutton-container">
```

Comment:
By convention, we prefer to add double quotes for all attributes: `style="Style"`